src/clj/oti/endpoint/virkailija.clj

```diff
@@ -141,6 +143,12 @@
 (defn- participant-routes [config]
   (routes
    (GET "/participant-search"   [q filter] (search-participant config q filter))
+   (GET "/diplomas" [id signer]
+     (let [ids (if (sequential? id) id [id])
+           id-set (->> (map utils/parse-int ids) (remove nil?) set)]
+       (if-not (or (empty? id-set) (str/blank? signer))
```

Comment:
Joidenkien mielestä on idiomaattisempaa käyttää `(if (seq coll) true false)` kuin `(if-not (empty coll) true false)`.. Ilmeisesti välttääkseen if-not emptyn tupla negaation.